interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1553,7 +1554,7 @@ def get_yaml(self, out):
         pass
 
 
-def convert(filename=None, output_name=None, text=None):
+def convert(filename=None, output_name=None, text=None, encoding=None):
```

Comment:
IMO, the default here should be `latin-1`. I see that `latin-1` is the default from the CLI, but I think it should be the default for the API as well.